api/src/main/java/org/openmrs/module/locationbasedaccess/AccessLocation.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * <p>
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.locationbasedaccess;
+
+import org.openmrs.PersonAttributeType;
+
+/**
+ * It will be used to store the personAttributeType for the location property and SessionLocationUuid of the user.
+ */
+public class AccessLocation {
```

Comment:
Introduced a new class which needs to carry the sessionLocationUuid, and personAttributeType for the location filtering.